tests/test_precovery_index.py

```diff
@@ -17,7 +17,7 @@ def test_precovery():
         out_dir="database",
         dataset_id="test_dataset",
         dataset_name="Test Dataset",
-        data_dir="tests/data",
+        data_dir="tests/data/index",
```

Comment:
Yeah, I have no problem making those changes. I'm trying to recall the reason I switched this up and I think it had something to do with scanning for files. Didn't want the sample orbits file getting pulled in along with the to-be-indexed observations file. Previously this didn't conflict because they had separate extensions and so the folder scanning didn't pick up the false positive when they lived in the same spot.